src/store/modules/user/actions.ts

```diff
@@ -74,6 +74,10 @@ const actions: ActionTree<UserState, RootState> = {
       })
 
       this.dispatch('util/getServiceStatusDesc')
+      await dispatch('getPinnedJobs', resp.data?.userLoginId).then((pinnedJobs: any) => {
```

Comment:
I think we should get pinned jobs on pipeline page instead of login, data will fetched once when the page is accessed.

Reply:
Sir, updated code.